content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,64 @@
+## Installation
```

Comment:
Ordering of categories is odd. Would have

General
Networking
Policy & Security 
Telemetry
Install & Tools
APIs & Configuration Management